pkg/config/testdata/containers_default.conf

```diff
@@ -261,6 +261,9 @@ runtime_supports_json = ["runc"]
 # SSH config file path
 ssh_config = "/foo/bar/.ssh/config"
 
+# Number of seconds to wait for Podmansh to timeout
```

Comment:
Wording: I think the desired outcome is for Podmansh to start, not to time out :)

Reply:
i'll change it to `podmansh login attempt to timeout`

Reply:
err... `Number of seconds to wait for podmansh login`. That should suffice I think.